controller/scan/registry.go

```diff
@@ -1472,16 +1485,21 @@ func (rs *Registry) scheduleScanImages(
 					sum.Status = api.ScanStatusScheduled
 					clusHelper.PutRegistryImageSummary(rs.config.Name, sum.ImageID, sum)
 				}
+
+				if signatureInfoChanged && sum.Status != api.ScanStatusScheduled {
+					skip = false
```

Comment:
When an image's digest doesn't change but nv sigstore settings change(assuming not scheduled yet), the image will be added to the scan task queue.
However, image digest is still the same which means vulnerability scan is not necessary if the image is never scanned(i.e only image signature verification is needed).
Does scanner have this mechanism to do image signature verification only?